db/db-setup.js

```diff
@@ -0,0 +1,18 @@
+require("dotenv").config()
+const mongoose = require("mongoose")
+
+const MONGODB_URI = process.env['MONGODB_URI']
+
+const initializeDbConnection = async () => {
+    try {
+        await mongoose.connect(MONGODB_URI, {
+            useNewUrlParser: true,
+            useUnifiedTopology: true
+        })
+        console.log("DB Connection Successful")
+    } catch (err) {
```

Comment:
you can console log the error as well to debug

Reply:
will do, thanks!